app/models/tag.rb

```diff
@@ -493,7 +493,7 @@ def self.autocomplete_fandom_lookup(options = {})
   def self.find_by_name(string)
     return unless string.is_a? String
     string = string.gsub('*s*', '/').gsub('*a*', '&').gsub('*d*', '.').gsub('*q*', '?').gsub('*h*', '#')
-    self.where('name = ?', string).first
+    self.where('tags.name = ?', string).first
```

Comment:
Yes, it can't tell that this is looking for `tags.name` rather than a field in `self`